data/mods/Magiclysm/items/spell_scrolls.json

```diff
@@ -662,6 +743,7 @@
     "type": "GENERIC",
     "copy-from": "spell_scroll",
     "id": "spell_scroll_lava_bomb",
+    "//": "Classless spell",
```

Comment:
if lava bomb is classless, that's a mistake, it should be earthshaper.

Reply:
![image](https://user-images.githubusercontent.com/39442864/75639590-8f77c280-5bff-11ea-9aa1-6a3da0749d32.png)

Reply:
not sure how to change that UI display but I changed my note and its new item_group

Reply:
oh, maybe the spell itself got missed then